support/Pipelines/Bbh/InitialData.py

```diff
@@ -12,6 +12,7 @@
 from spectre.Pipelines.EccentricityControl.InitialOrbitalParameters import (
     initial_orbital_parameters,
 )
+from spectre.support.DirectoryStructure import list_pipeline_steps
```

Comment:
Also import `PipelineStep` here (it's used below, this is why the test is currently failing)